host/cli/signal.go

```diff
@@ -0,0 +1,41 @@
+package cli
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/flynn/flynn/Godeps/_workspace/src/github.com/flynn/go-docopt"
+	"github.com/flynn/flynn/pkg/cluster"
+)
+
+func init() {
+	Register("signal", runSignal, `
+usage: flynn-host signal ID [SIGNAL]
+
+Signal a job`)
+}
+
+func runSignal(args *docopt.Args, client *cluster.Client) error {
+	id := args.String["ID"]
+	sig, err := strconv.Atoi(args.String["SIGNAL"])
+	if err != nil {
+		fmt.Println("invalid value for SIGNAL")
+		return err
+	}
+	hostID, err := cluster.ExtractHostID(id)
+	if err != nil {
+		fmt.Println("could not parse ", id)
```

Comment:
`fmt.Println` adds spaces between the arguments, so you can drop the trailing space here.